vignettes/plot_projection.Rmd

```diff
@@ -0,0 +1,283 @@
+---
+title: "Plot Model Projections Output"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.height = 2,
+  fig.width = 8,
+  fig.retina = 1,
+  fig.align = "center"
+)
+```
+
+The  `hubVis` package contains a function called 
+`plot_step_ahead_forecasts()` that can be used to plot model projections output.
+
+This function plots forecasts/scenario projections and optional truth data. 
+Faceted plots can be created for multiple scenarios, locations, 
+forecast dates, models, etc. 
+Currently, the function can plot only quantiles data, with the possibility to 
+add "median" information from the model projections.
+
+For more information about the Hubverse standard format, please refer to the
+[HubDocs website](https://hubdocs.readthedocs.io/en/latest/format/tasks.html).
+
+The following vignette describes the principal usage of the 
+`plot_step_ahead_forecasts()` function.
+
+```{r lib}
+library(hubVis)
+```
+
+Plots are available in two output formats:
+
+- "interactive" format: a [Plotly](https://plotly.com) output object with interactive legend, hover
+text, zoom-in and zoom-out options, etc.
+- "static" format: a [ggplot2](https://ggplot2.tidyverse.org/) output object.
+By default, the output plot is "interactive", but it can be changed to "static"
+by setting  the `interactive` parameter to FALSE. See end of the document for examples.
+
+## Load and Filter Data
+
+The package contains 2 datasets that can be used for examples:
+
+- `example_round1.csv`: example of model output for a round associated with the 
+origin date: "2021-03-07" (called "round 1"), target: "incident case", for the
+US national level, from the
+[example-complex-scenario-hub](https://github.com/Infectious-Disease-Modeling-Hubs/example-complex-scenario-hub). The data set also contains an ensemble calculated by applying the 
```

Comment:
These example files are probably ok as is, but I will mention that there was some discussion at some point about having more centralized example data files, so that we don't have to worry about updating all example data in all these random places when the format updates.

Reply:
I don't think we have done anything about this, but tagging @annakrystalli in case there is a better system in place anywhere that we could mimic. Currently the example projections and truth data file are stored as copies in this repo.